en/advanced/package_delivery.md

```diff
@@ -0,0 +1,49 @@
+# Package Delivery in Mission
```

Comment:
FYI I moved this here into development section. It will not longer be a target for other docs other than developers. I will pull out some of it for the package delivery intro in cargo.

Reply:
> I will pull out some of it for the package delivery intro in cargo.

What do you mean by that?

Reply:
What I mean is that if some of this is relevant for "end users" rather than "developers", I pull put it in the end user docs.